aboutus.css

```diff
@@ -0,0 +1,132 @@
+
+
+body {
```

Comment:
Same change requested as the html file. Transfer all styling to a .module.css file within an AboutUs Folder. :)